### PR TITLE
DOMA-2244 cypress base user flow

### PR DESCRIPTION
**Pull request by sitozzz**
This PR adds two base user scenarios:

**Auth scenario**: 
1. Login as an existing user
2. Register a new user
3. Reset password

**Ticket scenario**:
1. Create ticket
2. Filtering with global and local controls
3. Update ticket fields

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -6,6 +6,7 @@ COOKIE_SECRET=random
 SERVER_URL=http://localhost:3000
 DEFAULT_LOCALE=ru
 FILE_FIELD_ADAPTER=local
+GOOGLE_RECAPTCHA_CONFIG='{"SITE_KEY":"6LcPRvQaAAAAAJRyxsFIB4rP5VH036pFOkNH8lgh", "SERVER_KEY":"6LcPRvQaAAAAADn_h1440Es7fXIGD0E4lpXR_FyF"}'
```
[the remark this answers is not in the document]
Comment (dkoviazin) [reply .env.example:9]: Yes. Without reCaptcha we can not run e2e tests in browser

[the remark this answers is not in the document]
Comment (sitozzz) [reply .env.example:9]: > do we need to commit this?

Google recaptcha used at user restore and register scenario. So this key was configured to work with `localhost`.
Another way is save this value to github secret storage and then pass it to the workflow script.
But after discussing we decided to save this config as example string at environment file

File: apps/condo/bin/generate-initial-data.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+
+class InitialDataGenerator {
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
```
Comment (pahaz) [inline apps/condo/bin/generate-initial-data.js:7]: Could you please use prepareKeystoneApp from test utils

Comment (pahaz) [reply apps/condo/bin/generate-initial-data.js:7]: https://github.com/open-condo-software/condo/blob/master/apps/condo/bin/test-sbbol-oauth-by-hands.js#L22 Example

File: apps/condo/bin/generate-initial-data.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+    }
+}
+
+if (process.env.NODE_ENV === 'production') {
```
Comment (pahaz) [inline apps/condo/bin/generate-initial-data.js:14]: We use the production mod inside the CI

File: apps/condo/cypress.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (pahaz) [inline apps/condo/cypress.json:1]: Is it possible to move it inside cypress folder

File: apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx [hunk 5]
```diff
@@ -333,6 +333,7 @@ const ResetFiltersModalButton: React.FC<ResetFiltersModalButtonProps> = ({
             type={'text'}
             onClick={handleReset}
             size={size}
+            data-cy={'filters-button-reset'}
```
Comment (pahaz) [inline apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx:336]: It’s better to use something like ‘<domain>__<component name>—<element>’ format for such cases

File: apps/condo/bin/generate-initial-data.js (hunk 2 above)
Comment (sitozzz) [reply apps/condo/bin/generate-initial-data.js]: Done

File: apps/condo/cypress.json (hunk 4 above)
Comment (sitozzz) [reply apps/condo/cypress.json]: Move to cypress related dir

File: apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx (hunk 5 above)
Comment (sitozzz) [reply apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx]: Refactored this

**Pull request merged**